### PR TITLE
fix issue: add unit test for ReflectiveUtil.

### LINKED ISSUE #21746: Add unit test for ReflectiveUtil
This issue is to add unit tests for the ReflectiveUtil class and includes the following subtasks:

- [ ] ReflectiveUtil#getFieldValue
- [ ] ReflectiveUtil#setField
- [ ] ReflectiveUtil#setStaticField

### PR DESCRIPTION
**Pull request by pandaapo**
Fixes #21746.

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by pandaapo** [comment]
@sandynz Could you review this PR?

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.apache.shardingsphere.infra.util.reflect;
```
Comment (sandynz) [inline infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:1]: License header should be added

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class ReflectiveUtilTest {
```
Comment (sandynz) [inline infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:12]: `final` should be added for class

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Test
+    public void testGetFieldValue() {
```
Comment (sandynz) [inline infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:15]: Unit test method name should start with `assert`, e.g. assertGetFieldValue

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        UserPojo pojo = new UserPojo("test");
+        String fieldValue = (String)ReflectiveUtil.getFieldValue(pojo, "foo");
```
Comment (sandynz) [inline infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:17]: Blank is necessary after `(String)` for check style

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Test
+    public void testSetStaticField() {
+        ReflectiveUtil.setStaticField(UserPojo.class, "bar", "test");
+        assertTrue("test".equals(UserPojo.bar));
+    }
+
+    @AllArgsConstructor
+    @NoArgsConstructor
```
Comment (sandynz) [inline infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:35]: 1, If XxxArgsConstructor is not required, then they could be removed.

2, It's better to mark it as `private final` for inner class.

**Review by sandynz — CHANGES_REQUESTED** [review]
Thanks for you contribution.
Since you just submit PR today, suggest to read through [Set up env]( https://shardingsphere.apache.org/community/en/involved/contribute/dev-env/ ) and [Code of Conduct]( https://shardingsphere.apache.org/community/en/involved/conduct/code/ )

**Comment by pandaapo** [comment]
> Thanks for you contribution. Since you just submit PR today, suggest to read through [Set up env](https://shardingsphere.apache.org/community/en/involved/contribute/dev-env/) and [Code of Conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/)

Thanks for your guidance and review. I will check the contribute guide.

File: infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java (hunk 5 above)
Comment (pandaapo) [reply infra/util/src/test/java/org/apache/shardingsphere/infra/util/reflect/ReflectiveUtilTest.java:50]: The constructor with args was used in the test case.

**Comment by sandynz** [comment]
@pandaapo , I'm sorry, I didn't check the related issue yesterday, it's already assigned to another contributor. This PR could not be merged, could you try another issue? There's suggestion in original issue.

**Comment by pandaapo** [comment]
> @pandaapo , I'm sorry, I didn't check the related issue yesterday, it's already assigned to another contributor. This PR could not be merged, could you try another issue? There's suggestion in original issue.

I noticed that. But my PR was committed one day before the assigning.

**Comment by sandynz** [comment]
Hi @pandaapo , you could communicate it on the original issue. If there's new progress, I'll continue review.

**Comment by sandynz** [comment]
From the original issue, another issue is taken, so I close this PR.

**Pull request closed without merging**